### DISCUSSION TITLE
In Target Markdown interactive mode, print the return value if tar_simple is TRUE

### DISCUSSION THREAD

**Discussion by wlandau** [ropensci/targets#499] [Ideas]
## Prework

* [x] Read and agree to the [code of conduct](https://ropensci.org/code-of-conduct/) and [contributing guidelines](https://github.com/ropensci/targets/blob/main/CONTRIBUTING.md).
* [x] If there is [already a relevant issue](https://github.com/ropensci/targets/issues), whether open or closed, comment on the existing thread instead of posting a new issue.
* [x] For any problems you identify, post a [minimal reproducible example](https://www.tidyverse.org/help/) like [this one](https://github.com/ropensci/targets/issues/256#issuecomment-754229683) so the maintainer can troubleshoot. A reproducible example is:
    * [x] **Runnable**: post enough R code and data so any onlooker can create the error on their own computer.
    * [x] **Minimal**: reduce runtime wherever possible and remove complicated details that are irrelevant to the issue at hand.
    * [x] **Readable**: format your code according to the [tidyverse style guide](https://style.tidyverse.org/).

## Description

Should make it easier to prototype simple targets.

**Comment by wlandau** [comment]
Also print out "converted to target" later on in the output.

**Reply by wlandau** [reply]
Not actually sure if this is possible in `knitr` for a custom language engine.

**Comment by wlandau** [comment]
@cderv pointed out that users can simply switch back to the `{r}` engine when prototyping. So I think the `{targets}` engine can stick with the current behavior.

**Reply by jmbuhr** [reply]
oh, this is clever!
